tests/test_api.py

```diff
@@ -59,6 +59,9 @@ def _make_uds_server(path, request_handler):
     # Set daemon just in case something fails
     t.daemon = True
     t.start()
+
+    # Wait for the server to start
+    time.sleep(0.2)
```

Comment:
Can you use a e.g. `threading.Event` to sync the threads rather than sleeping?

Reply:
Hmm can't really because the thread going into an infinite loop in `serve_forever` so we can't hook into it directly. A slightly better approach would be to at least spin waiting until the server starts.

Reply:
^ I went ahead with that solution and it seems to work pretty well.